Core.DomainServices/Repositories/SSO/ISsoUserIdentityRepository.cs

```diff
@@ -10,5 +11,6 @@ public interface ISsoUserIdentityRepository
     {
         Maybe<SsoUserIdentity> GetByExternalUuid(Guid externalId);
         Result<SsoUserIdentity, OperationError> AddNew(User user, Guid externalId);
+        void DeleteIdentitiesForUser(IEnumerable<SsoUserIdentity> users);
```

Comment:
That is an odd argument list. That list could contain anything and is not related to a single user.
In stead pass the user instance and let this repo fetch the sso identities from the user instance